groceriesProject/src/main/java/com/malds/groceriesProject/repositories/clientRepository.java

```diff
@@ -0,0 +1,17 @@
+package com.malds.groceriesProject.repositories;
+
+import org.springframework.stereotype.Repository;
+import org.springframework.beans.factory.annotation.Autowired;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBMapper;
+import org.springframework.data.repository.CrudRepository;
+import org.socialsignin.spring.data.dynamodb.repository.EnableScan;
+
+import com.malds.groceriesProject.models.Client;
+
+//@EnableScan
+//public interface clientRepository extends CrudRepository<Client, Integer>{
+@Repository
+public class clientRepository{
```

Comment:
[nit] since you've made the repository a class, you should have the class name begin with a capital letter

Reply:
sounds good! Do u think changing it to a class is fine tho? most examples use interface

Reply:
Yeah it's fine. It's mostly a matter of discretion. I know that people use `interface` when they're extending `CrudRepository`(which is an interface available in Spring Boot containing a bunch of methods for CRUD operations). Otherwise, if you're writing your own crud methods to access a database (which is in our case), then you don't need the interface since you don't need to import `CrudRepository`.